test/audits/table-has-appropriate-headers-test.js

```diff
@@ -61,6 +61,19 @@ module('TableHasAppropriateHeaders');
         return table;
     }
 
+    test('Table with no child elements', function () {
```

Comment:
This is good, but we need a bunch more tests. At the very least, the original case we were trying to address, where we have a `<table>` with child elements but no `<tr>` descendants.